dark/reads.py

```diff
@@ -1512,6 +1513,37 @@ def variableSites(self, confirm=False, homogeneityLevel=1.0,
 
         return varSites
 
+    def combineReads(self):
+        """
+        Combine all reads into a single read.
+
+        @return: a C{str} sequence made from combining all reads.
+        """
+        reads = list(self)
+        assert len({len(read) for read in self.reads}) == 1
```

Comment:
If there's an assumption / requirement that all sequences must be the same length, you should say so in the docstring. Also in that case you don't need `zip_longest`...